airflow/providers/google/cloud/operators/cloud_build.py

```diff
@@ -583,6 +650,10 @@ def execute(self, context: 'Context'):
             timeout=self.timeout,
             metadata=self.metadata,
         )
+
+        CloudBuildListLink.persist(
```

Comment:
Can you please add indentation here for code readability?

Reply:
This code was automatically reformatted by pre-commit